src/database/seeds/DatabaseSeeder.php

```diff
@@ -11,6 +11,6 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
-        // $this->call(UserSeeder::class);
+        $this->call(CoronaStatusSeeder::class);
```

Comment:
```suggestion
        foreach ($this->getSeeders() as $seeder) {
            $this->call($seeder);
        }
```
Taip pat pridėk privatų metodą:
```
    private function getSeeders(): array
    {
        $seeders = [];

        $files = File::files(base_path('database/seeds'));

        foreach ($files as $file) {
            $filename = $file->getFilename();

            if (substr($filename, -10) !== 'Seeder.php') {
                continue;
            }

            $seeder = substr($filename, 0, -4);

            if ($seeder === self::class) {
                continue;
            }

            $seeders[] = $seeder;
        }

        return $seeders;
    }
```
Dabar nereikės kaskart kuriant seed'ą jį pridėti čia, kadangi visas seed'ų sąrašas bus nuskaitomas automatiškai.

Apie darbą su failais gali pasiskaityti čia: https://laravel.com/docs/8.x/filesystem